apps/docs/src/server.ts

```diff
@@ -1,3 +1,4 @@
+// eslint-disable-next-line @nx/enforce-module-boundaries
```

Comment:
Should we adjust the boundaries instead of disabling it here?

Reply:
The boundaries rule is failing due to the usage of a cjs `require` call somewhere else in the code:

```ts
const { createConfig } = require('@ng-rspack/build'); // here

module.exports = () => {
  if (global.NX_GRAPH_CREATION !== undefined) {
    return createConfig({
      browser: './src/main.ts',
      server: './src/main.server.ts',
      ssrEntry: './src/server.ts',
    });
  }
  return {};
};
```

The rule treats the library as lazy-loaded and throws "cannot directly import a lazy-loaded lib". I don't think we can adjust this behavior but I'm not completely sure. What could be the best approach here? IMO disabling the rule locally since it's used only in 2 places is the way to go.

Reply:
I was just wondering. I have no full overview of the codebase. I was just interested on the thought process as I do the same and always ask my self the same.